### ISSUE TITLE
svelte-flatpickr, localization, import object is undefined with nollup

### ISSUE THREAD

**Issue by frederikhors** [PepsRyuu/nollup#154]
_Originally opened here: https://github.com/rixo/svelte-template-hot/issues/8._

I'm trying to use [`svelte-flatpickr`](https://github.com/jacobmischka/svelte-flatpickr) which works very good with `npm run dev:rollup`.

Here a reproduction: https://github.com/frederikhors/svelte-template-hot/tree/svelte-flatpickr-localization.

This is the code: https://github.com/frederikhors/svelte-template-hot/blob/svelte-flatpickr-localization/src/FlatPickr.svelte:

I'm following the docs here: https://flatpickr.js.org/localization/.

```svelte
<script>
  import Flatpickr from "svelte-flatpickr";
  import { French } from "flatpickr/dist/l10n/fr.js";
  console.log("French", French); // why this is undefined?

  export let date = null;
  
  const options = {
    enableTime: true,
    locale: French,
    defaultDate: date,
    allowInput: true,
  };
</script>

<Flatpickr {options} bind:value={date} />
```

But this line: `import { French } from "flatpickr/dist/l10n/fr.js"` doesn't work: `French` is `undefined`.

**It works if I use this:**

```js
import French from "flatpickr/dist/l10n/fr.js";

...

locale: French.fr
```

**Why?**

**Comment by frederikhors** [comment]
**It works if I use this:**

```js
import French from "flatpickr/dist/l10n/fr.js";

...

locale: French.fr
```

**Comment by PepsRyuu** [comment]
Discovered the problem. Looks like syntheticNamedExports was updated to allow a string rather than just a boolean. https://rollupjs.org/guide/en/#synthetic-named-exports The CommonJS plugins exports to __moduleExports.

This can be easily supported. I have it prototyped already and have it working for the reproduction. I'll have it released at some point over the next few days once I test it further.

**Comment by PepsRyuu** [comment]
Released the fix in ```0.13.12```.
